src/middlewares/health.ts

```diff
@@ -1,20 +1,27 @@
 import { Context } from 'koa';
 import { isHealthy } from '../initializers/rabbitmq';
 
+export enum ConnectionStates {
+  disconnected = 0,
+  connected = 1,
+  connecting = 2,
+  disconnecting = 3,
+  uninitialized = 99
+}
+
 export default async function(ctx: Context, next: () => Promise<null>) {
-  const mongoose = await import('mongoose');
   const { getConnection } = await import('../initializers/mongodb');
 
   // tslint:disable-next-line: no-empty
   const mongoConnection = getConnection(() => {});
   // tslint:disable-next-line: no-empty
   const isRabbitHealthy = isHealthy();
-  if ((!mongoConnection || mongoConnection.readyState === mongoose.Connection.STATES.connected) && isRabbitHealthy) {
+  if ((!mongoConnection || mongoConnection.readyState === ConnectionStates.connected) && isRabbitHealthy) {
     const OrkaBuilder = (await import('../orka-builder')).default;
     ctx.status = 200;
     ctx.body = {
       env: OrkaBuilder.INSTANCE.config.nodeEnv,
-      version: `v${process.env.npm_package_version}`
+      version: process.env.DD_VERSION || `v${process.env.npm_package_version}`
```

Comment:
❓ why is DD_VERSION preferred to package.json's version?

Reply:
This is useful when we are on staging environments, where you will see something like `master.abc123` instead of `v1.2.1` which is not valid

Reply:
Ok, I don't like very much that it seems like a hack for our use case and not sth generic. But I am fine for now.

Reply:
Maybe we should ask SREs add another variable, called `VERSION` and use this one instead. WDYT?

Reply:
It would be better!